<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_Widget</class>
 <widget class="QWidget" name="Ui_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1921</width>
    <height>887</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>320</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Check The Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNameInput">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassInput">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>851</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="appName">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PassCheck</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;A &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Free and OpenSource &lt;/span&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Password Strength Checker. &lt;br/&gt;It uses real password patterns to mesure the strenght ot a given password.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passInfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Score : </string>
   </property>
  </widget>
  <widget class="QLabel" name="passInfo_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Guesses :</string>
   </property>
  </widget>
  <widget class="QLabel" name="passInfo_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>100 / hour:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passInfo_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>10k / second:</string>
   </property>
  </widget>
  <widget class="QLabel" name="passInfo_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>10B / second:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>170</y>
     <width>751</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="guesses">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>500</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tenH">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tenKS">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tenBS">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>660</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
